package/yast2-rear.changes

```diff
@@ -1,8 +1,16 @@
+-------------------------------------------------------------------
+Sat Jan 23 20:14:50 UTC 2021 - Petr Pavlu <petr.pavlu@suse.com>
+
+- Fix quoting of values in POST_RECOVERY_SCRIPT and
+  BACKUP_PROG_INCLUDE (bsc#1180983).
+- 4.2.3
```

Comment:
@petrpavlu, @jreidinger 

If I'm not wrong, it should be 4.3.0

Reply:
yes, as it differs now, it should be 4.3.0

Reply:
Updated to 4.3.0.